a10_neutron_lbaas/db/models.py

```diff
@@ -53,24 +59,47 @@ def find_all(cls, db_session=None):
         return cls.query(db_session).all()
 
     @classmethod
+    def create(cls, **kwargs):
+        instance = cls(**kwargs)
+        # Populate all the unspecified columns with their defaults
+        for key, column in inspect(cls).columns.items():
+            if key not in kwargs and column.default is not None:
+                arg = column.default.arg
+                column_default = arg if callable(arg) else lambda: arg
+                setattr(instance, key, column_default(instance))
+        return instance
+
+    @classmethod
     def create_and_save(cls, db_session=None, **kwargs):
         db = db_session or db_api.get_session()
-        m = cls(**kwargs)
+        m = cls.create(**kwargs)
         db.add(m)
         db.commit()
+        return m
 
     def as_dict(self):
         d = dict(self.__dict__)
         d.pop('_sa_instance_state', None)
         return d
 
+    def update(self, **kwargs):
+        for key in kwargs:
+            setattr(self, key, kwargs[key])
+
+    def delete(self, db_session=None):
+        db = db_session or inspect(self).session
+        db.delete(self)
+
+
+class A10BaseMixin(object):
+
     id = sa.Column(sa.String(36), primary_key=True, nullable=False, default=_uuid_str)
     tenant_id = sa.Column(sa.String(36), nullable=False)
     created_at = sa.Column(sa.DateTime, default=_get_date, nullable=False)
     updated_at = sa.Column(sa.DateTime, default=_get_date, onupdate=_get_date, nullable=False)
 
 
-class A10TenantBinding(A10BaseMixin, Base):
+class A10TenantBinding(A10BaseMixin, A10Base):
```

Comment:
I didn't remove the mixin. I moved all the model-agnostic code from it into A10Base. The mixin only has the four columns (id, tenant_id, created_at, updated_at_) now.